master/master.cfg

```diff
@@ -1438,7 +1432,7 @@ c['titleURL'] = 'http://halide-lang.org'
 # without some help.
 
 c['buildbotURL'] = 'https://buildbot.halide-lang.org/master/'
-# c['buildbotURL'] = 'http://buildbot.alexreinking.com:8012/'
+# c['buildbotURL'] = 'https://buildbot.alexreinking.com/'
```

Comment:
...do we want to leave this checked in?

Reply:
We can remove. I just find I need to keep changing those lines to make it work on my master :)